src/mito2/src/worker/handle_alter.rs

```diff
@@ -191,3 +212,92 @@ fn metadata_after_alteration(
+            let runs = parse_usize_with_default(key, value, default_option.max_active_window_runs)?;
+            log_option_update(region_id, key, options.max_active_window_runs, runs);
+            options.max_active_window_runs = runs;
+        }
+        mito_engine_options::TWCS_MAX_ACTIVE_WINDOW_FILES => {
+            let files =
+                parse_usize_with_default(key, value, default_option.max_active_window_files)?;
+            log_option_update(region_id, key, options.max_active_window_files, files);
+            options.max_active_window_files = files;
+        }
+        mito_engine_options::TWCS_MAX_INACTIVE_WINDOW_RUNS => {
+            let runs =
+                parse_usize_with_default(key, value, default_option.max_inactive_window_runs)?;
+            log_option_update(region_id, key, options.max_inactive_window_runs, runs);
+            options.max_inactive_window_runs = runs;
+        }
+        mito_engine_options::TWCS_MAX_INACTIVE_WINDOW_FILES => {
+            let files =
+                parse_usize_with_default(key, value, default_option.max_inactive_window_files)?;
+            log_option_update(region_id, key, options.max_inactive_window_files, files);
+            options.max_inactive_window_files = files;
+        }
+        mito_engine_options::TWCS_MAX_OUTPUT_FILE_SIZE => {
+            let size =
+                if value.is_empty() {
+                    default_option.max_output_file_size
+                } else {
+                    Some(ReadableSize::from_str(value).map_err(|_| {
+                        InvalidRegionOptionChangeRequestSnafu { key, value }.build()
+                    })?)
+                };
+            log_option_update(region_id, key, options.max_output_file_size, size);
+            options.max_output_file_size = size;
+        }
+        mito_engine_options::TWCS_TIME_WINDOW => {
+            let window =
+                if value.is_empty() {
+                    default_option.time_window
+                } else {
+                    Some(humantime::parse_duration(value).map_err(|_| {
+                        InvalidRegionOptionChangeRequestSnafu { key, value }.build()
+                    })?)
+                };
+            log_option_update(region_id, key, options.time_window, window);
+            options.time_window = window;
+        }
+        _ => return InvalidRegionOptionChangeRequestSnafu { key, value }.fail(),
+    }
+    Ok(())
+}
+
+fn parse_usize_with_default(
+    key: &str,
+    value: &str,
+    default: usize,
+) -> std::result::Result<usize, MetadataError> {
+    if value.is_empty() {
+        Ok(default)
+    } else {
+        value
+            .parse::<usize>()
+            .map_err(|_| InvalidRegionOptionChangeRequestSnafu { key, value }.build())
+    }
+}
+
+fn log_option_update<T: std::fmt::Debug>(
+    region_id: RegionId,
+    option_name: &str,
+    prev_value: T,
+    cur_value: T,
+) {
+    info!(
```

Comment:
```suggestion
    debug!(
```

Reply:
I think info is fine as change compaction option is not used often and we could stay consistent with ttl change's log level

Reply:
I see